src/schema/sources.ts

```diff
@@ -800,6 +841,7 @@ export const resolvers: IResolvers<any, Context> = {
         image,
         postId,
         description,
+        memberPostingRole = SourceMemberRoles.Member,
```

Comment:
Not sure if this is dangerous if for some reason the field doesn't get send it might change from admin to member again?

Reply:
Since this is on update it will fail only if `null` is sent. By default (if nothing is sent for BC purposes it will fallback to member due to this).

Reply:
I can add a tests to cover the cases?

Reply:
That's my concern.

Let's say someone set it to "admin only" and on a next update we only send the title for edit.
Now it will be reset to member right?

Reply:
This mutation is for create squad only right? The logic will be different for `editSquad`, this is not yet implemented, will do it in scope of another task. Added some tests for create here c9b7215cb46ec7c63b955f4c559c1e981432599a

Reply:
Ah yes sorry, the GitHub PR showed "EditSquadArgs" above so kind of assumed it was a generic one.
But yes in that case never mind my comment.